src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -127,6 +127,7 @@ const PatientsAppNav = View.extend({
     search: '.js-search',
   },
   onSearchActive(isActive) {
+    if (this.isDestroyed()) return;
```

Comment:
In this case the nav was switching away from the patients nav, and then the search modal was destroying which triggered this event causing an error.